### ISSUE TITLE
Source for Stevenson-Arce?

### ISSUE THREAD

**Issue by makeworld-the-better-one** [hbldh/hitherdither#6]
hitherdither has a matrix for the Stevenson-Arce error diffusion dither, as see here:

https://github.com/hbldh/hitherdither/blob/27c939273bdc8549e0f18b8b5ddc469266638bdf/hitherdither/diffusion.py#L97-L111

As noted [here](https://en.wikipedia.org/wiki/Talk%3ADither#Removal_of_original_research):

> Stevenson-Arce is designed to work on hexagonal arrays and will _obviously_ give ugly results on regular square pixel images.

And [this source](https://bisqwit.iki.fi/jutut/kuvat/ordered_dither/error_diffusion.txt) describes Stevenson-Arce like this:

```
Stevenson-Arce:
	
	            *   .  32
	12  .   26  .  30   . 16
        .   12  .  26  .   12  .
	5   .   12  .  12   .  5    / 200
```

This obviously shows a hexagonal arrangement.

So where did you get your numbers from, or how did you derive them? Do you have any example pictures of what this looks like?

**Comment by hbldh** [comment]
I think I found the [same reference](https://bisqwit.iki.fi/jutut/kuvat/ordered_dither/error_diffusion.txt) as you did (see the [docstring to the method `error_diffusion_dithering`](https://github.com/hbldh/hitherdither/blob/27c939273bdc8549e0f18b8b5ddc469266638bdf/hitherdither/diffusion.py#L115-L179)) and just threw it in there with all the others. It also seems like I made an error in copying it. I haven't tested its performance more than making sure that it can be run, as is clearly stated in the above mentioned docstring.

Send me a PR removing it and I will merge it.
